tools/external_ic.F90

```diff
@@ -4162,29 +4166,34 @@ subroutine d2a3d(u, v,  ua,   va,  im,  jm, km, lon)
 
   end subroutine d2a3d
 
-
-
   subroutine pmaxmin( qname, a, im, jm, fac )
 
       integer, intent(in):: im, jm
       character(len=*) :: qname
       integer i, j
-      real a(im,jm)
-
-      real qmin(jm), qmax(jm)
-      real pmax, pmin
-      real fac                     ! multiplication factor
-
-      do j=1,jm
-         pmax = a(1,j)
-         pmin = a(1,j)
-         do i=2,im
-            pmax = max(pmax, a(i,j))
-            pmin = min(pmin, a(i,j))
+      class(*) a(im,jm)
```

Comment:
Why do you need to update pmaxmin?

Reply:
to elaborate, it doesn't seem that any of the calls to pmaxmin require mixed-mode usage.

Reply:
It is because the variable `sst_ncep` is sometimes used as an argument to `pmaxmin`. The `sst_ncep` variable can be coming from the `FMS/amip_interp/amip_interp.F90` as opposed to `atmos_cubed_sphere/tools/external_sst.F90`. @binli2337 Could you confirm?